README.md

```diff
@@ -44,7 +44,9 @@ You can install SuperSuit via `pip install supersuit`
 
 `sticky_actions_v0(env, repeat_action_probability)` assigns a probability of an old action "sticking" to the environment and not updating as requested. This is to prevent agents from learning predefined action patterns in highly deterministic games like Atari. Note that the stickiness is cumulative, so an action has a repeat_action_probability^2 chance of an action sticking for two turns in a row, etc. This is the recommended way of adding randomness to Atari by *"Machado et al. (2018), "Revisiting the Arcade Learning Environment: Evaluation Protocols and Open Problems for General Agents"*
 
-`frame_stack_v1(env, num_frames=4)` stacks the most recent frames. For vector games observed via plain vectors (1D arrays), the output is just concatenated to a longer 1D array. 2D or 3D arrays are stacked to be taller 3D arrays. At the start of the game, frames that don't yet exist are filled with 0s. `num_frames=1` is analogous to not using this function.
+`frame_stack_v1(env, num_frames=4, stack_dim0=False)` stacks the most recent frames. For vector games observed via plain vectors (1D arrays), the output is just concatenated to a longer 1D array. 2D or 3D arrays are stacked to be taller 3D arrays. Stacked dimension can be set to dim=0 (default dim=-1) by stack_dim0=True. At the start of the game, frames that don't yet exist are filled with 0s. `num_frames=1` is analogous to not using this function.
+
+`frame_stack_v2(env, num_frames=4, stack_dim0=False)` stacks the most recent frames. For vector games observed via plain vectors (1D arrays), the output is just concatenated to a longer 1D array. 2D or 3D arrays are stacked to be taller 3D arrays. Stacked dimension can be set to dim=0 (default dim=-1) with `stack_dim0=True`. At the start of the game, frames that don't yet exist are filled with the copies of the first frame. `num_frames=1` is analogous to not using this function.
```

Comment:
While I'm sure that it works, can we change `stack_dim0` to just become `stack_dim`, and then assert it to be either 0 or -1? This way if there are new API changes in the future that allow for arbitrary stacking dimensions, the arguments between the new API and this one won't be any different.

Also, do keep arguments (like `stack_dim0=True`) inside code lines.